index.html

```diff
@@ -10,392 +10,422 @@
+          <h1 class="text -x-small -color-dark -caps -wide-spacing -bold -centered">
+            A Viget Exploration
+            <img class="header__img" src="./public/img/title.png" alt="Improving Medical Adherence">
+          </h1>
+
+          <div class="pills -yellow"></div>
+          <div class="pills -multiple"></div>
+          <div class="pills -double"></div>
+          <div class="pills -purple"></div>
+        </div>
+      </header>
+
+
+      <!--INTRO SECTION-->
+      <section class="intro bg-white">
+        <div class="content-wrap lrg-rl-padding">
           <p class="text -medium -serif">Patients who don’t take medication as prescribed are at risk of worsening health, increased hospitalization, and a greater chance of mortality.</p>
           <p class="text -small">Just in the U.S., non-adherence has been attributed to over $100 billion in avoidable healthcare costs.<cite><a href="">1</a></cite> In developed countries, among patients with chronic illness, approximately 50% do not take medications as prescribed.<cite><a href="">2</a></cite></p>
 
-          <div class="quote-container">
+          <blockquote class="quote-container">
             <div class="pills bottle-1"></div>
             <div class="pills bottle-2"></div>
             <div class="pills bottle-3"></div>
             <div class="quote">
               <p class="text -small -serif -color-accent">“Increasing the effectiveness of adherence interventions might have a greater impact on the health of the population than any improvement in specific medical treatment.”</p>
-              <footer class="text -x-small -caps -bold">World Health Organization, 2003<cite><a href="">3</a></cite></footer>
+              <span class="text -x-small -caps -bold">World Health Organization, 2003<cite><a href="">3</a></cite></span>
             </div>
-          </div>
-        </section>
-    </div>
+          </blockquote>
+        </div>
+      </section>
 
-    <!--DIAGRAM SECTION-->
-    <div class="content-wrap">
+      <!--DIAGRAM SECTION-->
       <section class="diagram">
-        <div class="centered">
-          <p class="text -x-small -color-dark -caps -wide-spacing -bold">Why People Don't Adhere</p>
-          <p class="text -large -serif -color-dark">Myriad factors conspire to prevent people from sticking to their treatment programs. But a few are most telling.</p>
-        </div>
-        <div class="diagram-details">
-          <div class="diagram-detail centered">
-            <!--transparent borders-->
-            <div class="icon -borders"></div>
-            <!--background-color-->
-            <div class="icon -color">
-              <!--img-->
-              <img class="resize" src="./public/img/icon-algorithmic.png">
+        <div class="content-wrap">
+          <!--Diagram heading-->
+          <div class="heading-container">
+            <h2 class="text -x-small -color-dark -caps -wide-spacing -bold">Why People Don't Adhere</h2>
```

Comment:
This is cool but like I said in the scss file you could make `caps` and `bold` be more generic and not just modifier classes. That way you could use them on anything without having to write the rules for caps and bold every time.

Alternatively if you aren't going to need those atomic classes you can condense all those modifiers into 1 rule:

eg. `sub-heading -pre-title`

where `sub-heading` has all your rules and then `-pre-title` (or whatever) adds the rules to make it all caps, have wide spacing, make it color-dark, and make it bold.